JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -108,7 +108,9 @@ private async Task<bool> CallMessageHandler(Message message)
 
             if (handler == null)
             {
-                return true;
+                _log.LogError("Handler for message of type '{MessageTypeName}' not found in handler map. Returning message to queue.", message.GetType().FullName);
```

Comment:
Should we have a test to test that false is returned from the dispatcher when there is no handler for it?

Reply:
This is implied, by the test we already have, where it asserts `ShouldUpdateMessageVisibility()`, as it was already done for the `AndMessageProcessingFails` scenario.

Reply:
I guess I don't quite understand why we're asserting that a side effect occurred, instead of directly asserting that the message dispatcher returned false. IMO we shouldn't need an SQS client to test the message dispatcher.

Reply:
Ah ok I can see I've misunderstood a bit here - it is the only way to determine that as MessageDispatcher is a bit of a black box and doesn't return. I was confusing it with the handler 🙂. Fair go, thanks for clarifying 👍

Reply:
I've added a comment here:
https://github.com/justeat/JustSaying/pull/728#discussion_r449260954